pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -99,6 +112,10 @@ func (s *Service) GetForProviderWithRedactedSecrets(ctx context.Context, provide
 		return nil, ssosettings.ErrNotConfigurable
 	}
 
+	if provider == social.SAMLProviderName {
```

Comment:
I think it's not needed. If the provider is configurable then it should be returned.

Reply:
Can you please explain why this check is needed?

Reply:
This check has been removed. The logic for this is already included in the `ProvideService` function.